xds/src/test/java/io/grpc/xds/XdsClientImplTestBase.java

```diff
@@ -3516,7 +3516,7 @@ public void sendingToStoppedServer() throws Exception {
       // Establish the adsStream object
       xdsClient.watchXdsResource(XdsClusterResource.getInstance(), CDS_RESOURCE,
           cdsResourceWatcher);
-      resourceDiscoveryCalls.take(); // clear this entry
+      var unused = resourceDiscoveryCalls.take(); // clear this entry
```

Comment:
The `var` keyword was used in Java10 whereas we are supposed to be Java8 compliant?

Reply:
Sorry. FIxed.